src/pages/Game/Game.tsx

```diff
@@ -0,0 +1,150 @@
+/* eslint-disable no-self-compare */
+import React, { useState, useEffect, useMemo } from "react";
+import { ToastContainer, toast } from "react-toastify";
+import { useSelector, useDispatch } from "react-redux";
+import { useNavigate } from "react-router-dom";
+//images
+import bomber from "images/img-game/13026.png";
+import smileNotRight from "images/img-game/smile-sad.png";
+//Components
+import Button from "сomponents/Button";
+//redux state
+import { RootState } from "redux/store";
+//reducer
+import { makeMove } from "redux/store";
```

Comment:
u can combine line 12 and 14 as import { makeMove, RootState } from "redux/store";